src/SimpleAzure.Storage.HybridQueue/HybridQueue.cs

```diff
@@ -186,73 +157,65 @@ public async Task DeleteMessageAsync<T>(HybridMessage<T> hybridMessage, Cancella
 
         _logger.LogDebug("About to receive queue message.");
 
-        var response = await _queueClient.ReceiveMessagesAsync(maxMessages, visibilityTimeout, cancellationToken);
-
-        if (response == null ||
-            response.Value == null)
+        var response = await _queueClient.ReceiveMessagesAsync(maxMessages, visibilityTimeout, cancellationToken).ConfigureAwait(false);
+        if (response?.Value is not { } messages)

```

Comment:
again, i don't understand and need to learn this magic...

Reply:
These do take a little time to get used to, I'll admit